tests/system/test_system.py

```diff
@@ -353,16 +353,34 @@ def setUpClass(cls):
             SPANNER_OPERATION_TIMEOUT_IN_SECONDS
         )  # raises on failure / timeout.
 
+        # Create a multi-region instance
+        multi_region_config = "nam3"
+        ALT_INSTANCE_ID = "new" + unique_resource_id("-")
```

Comment:
nit: All caps are usually used for global/static constants. Move this to the top with `INSTANCE_ID` so it's clear what format the instance IDs are generated. Additionally, rather than `new` perhaps `multi-region` would be better. That way it's clear what the instance is for.